step.yml

```diff
@@ -12,6 +12,9 @@ support_url: https://github.com/bitrise-steplib/bitrise-step-activate-gradle-rem
 type_tags:
 - utility
 
+project_type_tags:
+- android
```

Comment:
I think we want to target Flutter and React Native too

Reply:
Are those supported / tested?

Reply:
Yes, both of them are supported because the Android side of a RN/Flutter project is a plain Gradle project. Maybe some Flutter/RN specific tasks are marked as non-cacheable, but the rest of the common build logic should benefit from caching.